feugy/tabulous
# feat(web): introduce table texture

### What's in there?

White tables are so boring... Let's use some textures

Are included here:

- feat(server): adds table spec to game descriptor
- feat(web): introduces some table textures
- feat(web): supports color and image textures for table
- feat(klondike): uses a better board and table texture
- feat(coinche): uses a table texture
- fix(web): no stack indicators on handless games
- refactor(klondike): aligns logic with latest standard
- refactor(coinche): aligns logic with latest standard

### How to test?

Try out Klondike and Coinche to see their new table texture.
The indicator bug is easily reproducible: Klondike has no indicators, while it should.
